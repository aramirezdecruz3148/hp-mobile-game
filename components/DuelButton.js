import React from "react";
import { View, Button } from "react-native";
import PropTypes from "prop-types";

export default function DuelButton({ setDuel }) {
  return (
    <View>
      <Button 
        title="Duel"
        onPress={setDuel}
      />
    </View>
  );
}

DuelButton.propTypes = {
  setDuel: PropTypes.func.isRequired
}