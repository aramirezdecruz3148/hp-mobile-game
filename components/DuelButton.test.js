import React from 'react';
import { shallow } from 'enzyme';
import DuelButton from './DuelButton';

describe('DuelButton component', () => {
  it('renders DuelButton', () => {
    const wrapper = shallow(<DuelButton 
      setDuel={() => { }}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});