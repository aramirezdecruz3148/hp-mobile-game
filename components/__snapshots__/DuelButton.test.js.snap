// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DuelButton component renders DuelButton 1`] = `
<View>
  <Button
    onPress={[Function]}
    title="Duel"
  />
</View>
`;
